1.adoc

```diff
@@ -23,6 +23,37 @@ Platforma:: Webový prohlížeč (editor je komponenta použitelná v rámci web
 == Product statement
 == Business requirements
 == Use cases
+
+. **Zvolitelný zobrazovací mód editoru**
++
+Uživatel může zvolit zobrazení editoru vhodné pro to, čemu se chce právě věnovat.
++
+ - Dvousloupčekový mód editoru (zdrojový kód a náhled) je vhodný pro běžné úpravy, nebo pokud uživatel využívá pokročilé funkce LML a chce zkontrolovat výstup preprocessoru.
+ - Náhled pouze zdrojového kódu je vhodný pro nerušené psaní dokumentu, kde se autor soustředí na obsah.
+ - Náhled pouze dokumentu je vhodný pro revizi hotového obsahu.
+
+. **Editace dokumentu**
++
+Uživatel upravuje dokument přímou editací zdrojového kódu v LML.
+
+. **Navigace v dokumentu**
++
+Každý LML vytváří implicitně strukturu dokumentu, pomocí nadpisů různých úrovní.
+U větších dokumentů je vhodné využít tuto strukturu pro navigaci ve zdrojovém kódu.
+
+. **Nastaveni voleb editoru**
++
+Uživatel musí mít možnost nastavení funkcí editoru, které nesouvisí přímo s editací zdrojového kódu dokumentu.
+Například se jedná o přepínání zobrazovacích módu.
+
+. **Pomoc uživateli s pokročilými funkcemi**
++
+Editor bude poskytovat uživatelům přirozené UI pro úpravu dokumentu.
+Vzhledem k povaze uživatelů se předpokládá, že preferují ovládání pomocí klávesnice před klikáním na popsaná tlačítka.
+Bylo by vhodné poskytnout nějakou formu pomoci, tutorialu, či přehledu pokročilých funkcí editoru.
+
+
```

Comment:
Jak moc detailní use-cases chceme?

Např.

**Vložení obrázku**

Uživatel má možnost vložit do dokumentu obrázek způsobem který mu nejvíce vyhovuje. Např. ruční zapsání adresy pomocí LML, využití UI dialogu ("vybrat soubor"), či přetažení/pastnutí obrázku přímo do dokumentu.

Reply:
Po diskusi s Hunkou je tohle uz task, nikoli use case.

Reply:
ok, pridam tedy do tasku